include/mxnet/c_api.h

```diff
@@ -227,10 +231,132 @@ MXNET_DLL int MXSetProfilerConfig(int mode, const char* filename);
  */
 MXNET_DLL int MXSetProfilerState(int state);
 
-/*! \brief Save profile and stop profiler */
+/*!
+ * \brief Save profile and stop profiler
+ * \param append true if appending to current profile file, false for truncate
+ * \return 0 when success, -1 when failure happens.
+ */
 MXNET_DLL int MXDumpProfile();
 
-/*! \brief Set the number of OMP threads to use */
+
+/*!
+ * \brief Print aggregate stats to the console
+ * \param reset Clear the aggregate stats after printing
+ * \return 0 when success, -1 when failure happens.
+ * \note
+ */
+MXNET_DLL int MXDumpAggregateProfileStats(int reset);
```

Comment:
Would it be better return as string?

Reply:
you want to return the humongous output text?  Currently just dumps to console.  Is that what pytorch does -- I didnt check?